draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -65,38 +64,39 @@ Additionally, the confidentiality and integrity mechanisms in RADIUS rely on the
 Although RADIUS/(D)TLS does not remove the MD5-based mechanisms, it adds confidentiality and integrity protection through the TLS layer.
 For an updated version of RADIUS/(D)TLS without need for MD5 see {{?RFC9765}}.
 
+This specification largely preserves the format and processing of RADIUS packets, as defined in {{RFC2865}}, {{RFC2866}} and {{RFC5176}}.
+However, there are some minor changes required to RADIUS packet processing, which are discussed in {{radiuschanges}}.
+
 # Conventions and Terminology
 
-{::boilerplate bcp14-tagged}
+RadSec:
+: A collective term for RADIUS/TLS and RADIUS/DTLS.
 
 The following terminology is used in this document:
 
-RadSec:
-: A collective term for RADIUS/TLS and RADIUS/DTLS.
+RADIUS packet:
+: As defined in {{RFC2865}}.
+
+RADIUS packet type:
+: As defined in {{RFC2865}}.
 
 RADIUS/TLS:
 : A RADIUS exchange transmitted using TLS over TCP.
 
 RADIUS/DTLS:
 : A RADIUS exchange transmitted using DTLS over UDP.
 
+RadSec:
+: A collective term for RADIUS/TLS and RADIUS/DTLS.
+
 RADIUS/UDP:
 : RADIUS transported over UDP as defined in {{RFC2865}}.
 
-RADIUS packet:
-: As defined in {{RFC2865}}.
-
-RADIUS packet type:
-: As defined in {{RFC2865}}.
-
 (D)TLS handshake message:
 : As defined in TLS {{RFC5246}} and DTLS {{RFC9147}}.
 
-TLS record:
-: As defined in TLS {{RFC5246}}.
-
-DTLS record:
-: As defined in DTLS {{RFC9147}}.  A DTLS record is always contained in one UDP datagram.
+(D)TLS record:
+: As defined in DTLS {{RFC9147}}. A DTLS record is always contained in one UDP datagram.
```

Comment:
I think TLS and DLTS records are slightly different and specified in different RFCs. I propose to keep the original definitions.